daemon/hostdaemon.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"github.com/go-logr/logr"
+	"fmt"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+)
+
+type HostDaemon struct {
+	dev bool
+	log logr.Logger
+	client pb.BridgePortServiceClient
+	vsp VendorPlugin
+}
+
+func (d *HostDaemon) CreateBridgePort(pf int, vf int, vlan int) (error) {
+	createRequest := &pb.CreateBridgePortRequest{
+		BridgePort: &pb.BridgePort{
+			Name: fmt.Sprintf("%d-%d-%d", pf, vf, vlan),
+			Spec: &pb.BridgePortSpec{
+				Ptype:          1,
+				MacAddress:     []byte{},
+				LogicalBridges: []string{},
+			},
+		},
+	}
+
+	_, err := d.client.CreateBridgePort(context.TODO(), createRequest)
+	return err
+}
+
+func (d *HostDaemon) DeleteBridgePort(pf int, vf int, vlan int) (error) {
+	req := &pb.DeleteBridgePortRequest{
+		    Name: fmt.Sprintf("%d-%d-%d", pf, vf, vlan),
+	}
+
+	_, err := d.client.DeleteBridgePort(context.TODO(), req)
+	return err
+}
+
+func NewHostDaemon(dev bool, vsp VendorPlugin) *HostDaemon {
```

Comment:
shouldn't this be `vsp *VendorPlugin`? From `NewDummyPlugin(mode string) *DummyPlugin`